spec/form_objects/forms/payment_link_form_spec.rb

```diff
@@ -2,59 +2,30 @@
 
 RSpec.describe Forms::PaymentLinkForm, type: :model do
   describe "Payment URL" do
-    context "when form is live" do
-      let(:form) do
-        build(:form, :live)
-      end
-
-      it "validates the URL" do
-        payment_link_form = described_class.new(form:, payment_url: "gov.uk")
-        error_message = I18n.t("errors.messages.url")
-
-        payment_link_form.validate(:payment_url)
-
-        expect(payment_link_form.errors.full_messages_for(:payment_url)).to include(
-          "Payment url #{error_message}",
-        )
-      end
-
-      it "is valid if blank" do
-        payment_link_form = described_class.new(form:, payment_url: "")
-
-        expect(payment_link_form).to be_valid
-      end
-
-      it "is valid if a PAY url is given" do
-        payment_link_form = described_class.new(form:, payment_url: "https://www.gov.uk/payments/test-org/test-service")
-
-        expect(payment_link_form).to be_valid
-      end
+    let(:form) do
+      build(:form, :live)
     end
 
-    context "when form is not live" do
-      let(:form) do
-        build(:form)
-      end
+    it "validates the URL" do
```

Comment:
This spec declaration is confusing. The example used _isn't_ valid, so it isn't being validated... Maybe something like
``` 
context "is a Pay link"
  it "validates"
...
context "not a Pay link"
  it "fails validation"
```

Reply:
fixed, context specified for the tests